drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -87,6 +88,19 @@ def household_size
       household_members.size
     end
 
+    def hud_chronic
+      # HUD Chronic status for the client that was referred as HoH
+      referred_hoh_client = hoh_member&.client
+      return unless referred_hoh_client.present?
```

Comment:
doesn't hurt but `present` isn't necessary since relationship is either be nil or an object

Reply:
ok, thanks. I am so in the habit of using it everywhere! 🙈